raiden/network/rpc/client.py

```diff
@@ -772,6 +772,9 @@ def deposit(self, our_address, amount):  # pylint: disable=unused-argument
 
         log.info('deposit called', contract=pex(self.address), amount=amount)
 
+    def transferredAmount(self, participant_address):
```

Comment:
hummm, I find it weird, but if the rest agreed with it I'm fine, let's just open a issue to rename it all.